src/Symfony/Bundle/SecurityBundle/DependencyInjection/Compiler/AddSecurityVotersPass.php

```diff
@@ -45,7 +46,7 @@ public function process(ContainerBuilder $container)
             throw new LogicException('No security voters found. You need to tag at least one with "security.voter"');
         }
 
-        $adm = $container->getDefinition($container->hasDefinition('debug.security.access.decision_manager') ? 'debug.security.access.decision_manager' : 'security.access.decision_manager');
-        $adm->addMethodCall('setVoters', array(array_values($voters)));
+        $adm = $container->getDefinition('security.access.decision_manager');
```

Comment:
IIRC we explicitly have this handling because there were issue otherwise when you were, for example, using the JMSSecurityExtraBundle which ships with its own [RememberingAccessDecisionManager](https://github.com/schmittjoh/JMSSecurityExtraBundle/blob/master/Security/Authorization/RememberingAccessDecisionManager.php).